### PR TITLE
Move SDP manipulating url params parsing to client

### PR DESCRIPTION
**Pull request by jiayliu**
Previously GAE parses the url params and simply passes them back to the JS, which adds no value.

Now the SDP manipulating params are completely handled on the client side.

### CODE REVIEW

**Comment by jiayliu** [comment]
@juberti @tkchin

File: samples/web/content/apprtc/js/appcontroller.js [hunk 1]
```diff
@@ -470,6 +472,25 @@ AppController.prototype.showIcons_ = function() {
   }
 };
 
+AppController.prototype.loadUrlParams_ = function() {
+  /* jshint ignore:start */
+  var queries = urlQueryToDictionary();
```
Comment (juberti) [inline samples/web/content/apprtc/js/appcontroller.js]: |queries| would probably be better named |urlParams|

File: samples/web/content/apprtc/js/appcontroller.js [hunk 2]
```diff
@@ -470,6 +472,25 @@ AppController.prototype.showIcons_ = function() {
   }
 };
 
+AppController.prototype.loadUrlParams_ = function() {
+  /* jshint ignore:start */
```
Comment (juberti) [inline samples/web/content/apprtc/js/appcontroller.js]: Why is this needed?

File: samples/web/content/apprtc/js/util.js [hunk 3]
```diff
@@ -18,6 +19,18 @@ function $(selector) {
...
 }
 
+// Returns the URL query key-value pairs as a dictionary object.
+function urlQueryToDictionary() {
+  var pairs = location.search.slice(1).split('&');
+
+  var result = {};
+  pairs.forEach(function(pair) {
+    pair = pair.split('=');
+    result[pair[0]] = decodeURIComponent(pair[1] || '');
```
Comment (juberti) [inline samples/web/content/apprtc/js/util.js]: should the default value be |true| instead of '', similar to HTML attributes?

File: samples/web/content/apprtc/js/util.js [hunk 4]
```diff
@@ -18,6 +19,18 @@ function $(selector) {
   return document.querySelector(selector);
 }
 
+// Returns the URL query key-value pairs as a dictionary object.
+function urlQueryToDictionary() {
```
Comment (juberti) [inline samples/web/content/apprtc/js/util.js]: queryStringToDictionary

**Comment by juberti** [comment]
nice cleanup, lgtm with nits above

**Comment by chuckhays** [comment]
LGTM as well, with the same name comments as juberti.

File: samples/web/content/apprtc/js/util.js (hunk 3 above)
Comment (jiayliu) [inline samples/web/content/apprtc/js/util.js]: This is to preserve the original GAE logic which uses '' as the default.

File: samples/web/content/apprtc/js/appcontroller.js (hunk 2 above)
Comment (jiayliu) [inline samples/web/content/apprtc/js/appcontroller.js]: Added comment to explain.

**Comment by tkchin** [comment]
lgtm

**Comment by juberti** [comment]
lgtm
